bucket/edgedb.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.0.2+61917d6",
+    "description": "A graph-relational database",
+    "homepage": "https://www.edgedb.com/",
+    "license": "Apache-2.0|MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://packages.edgedb.com/archive/x86_64-pc-windows-msvc/edgedb-cli-2.0.2+61917d6.exe#/edgedb.exe",
+            "hash": "0f8cfe8b4c8b03abb2a1dcc6b540a3ec43b1fd728b8757d5f86df1c469f308fa"
+        }
+    },
+    "bin": "edgedb.exe",
+    "checkver": {
+        "url": "https://packages.edgedb.com/archive/.jsonindexes/x86_64-pc-windows-msvc.json",
+        "jsonpath": "$..packages[-1:].version"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://packages.edgedb.com/archive/x86_64-pc-windows-msvc/edgedb-cli-$version.exe#/edgedb.exe"
+            }
+        },
+        "hash": {
+            "url": "https://packages.edgedb.com/archive/.jsonindexes/x86_64-pc-windows-msvc.json",
```

Comment:
This URL seems x86_64 specific, so shouldn't it go inside the architecture block?

Reply:
It's sometimes inside and sometimes it is not. :smile: Examples: 
https://github.com/ScoopInstaller/Main/blob/master/bucket/containerd.json
https://github.com/ScoopInstaller/Main/blob/master/bucket/chronograf.json

Reply:
Ah yes, pre-existing inconsistencies are there 😅

But I try to put architecture-specific URLs inside, and generic URLs outside, as a convention.